dev-kubernetes-manifests/ledger-writer.yaml

```diff
@@ -70,11 +70,18 @@ spec:
             memory: 1Gi
         readinessProbe:
           httpGet:
-            path: /ready
+            path: /z/health/readiness
```

Comment:
I chose `/z` because things like `/admin` and `/management` have too much of a potential for collision. The `/health` endpoint pattern is known as "healthz" endpoints, so it was a play on that. Unfortunately, it doesn't align with any known convention and is my thing for lack of a better idea.